openvdb_houdini/houdini/SOP_OpenVDB_Scatter.cc

```diff
@@ -447,13 +505,17 @@ SOP_OpenVDB_Scatter::cookMySop(OP_Context& context)
 
             // add ALL the points to this group
             ptgroup->addRange(gdp->getPointRange());
-        }
 
-        // add the VDBs to the output if requested
-        if (1 == evalInt("keep", 0, time)) {
-            gdp->mergePrimitives(*vdbgeo, vdbgeo->getPrimitiveRange());
+            const std::string groupName(scatterStr.toStdString());
+            for (auto& pointGrid : pointGrids) {
+                openvdb::points::appendGroup(pointGrid->tree(), groupName);
+                openvdb::points::setGroup(pointGrid->tree(), groupName);
```

Comment:
This seems pretty confusing to me. The whole point of this mode is to differentiate points that have just been generated from other points merged in to the GDP from upstream using a group, that really doesn't apply to VDB Points, so I'd suggest perhaps hiding this option from the UI entirely when scattering VDB Points.

Reply:
What I really like above this implementation is that currently every option on the scatter sop works for VDB points. Whilst I agree this makes the most sense for Houdini points I don't think it's that big a stretch or too confusing just to allow VDB Points to be assigned a uniform group.

Reply:
This is true, fair enough.